sdk/identity/identity/src/credentials/managedIdentityCredential/models.ts

```diff
@@ -14,9 +14,12 @@ export interface MSI {
     getTokenOptions?: GetTokenOptions
   ): Promise<boolean>;
   getToken(
-    identityClient: IdentityClient,
-    resource: string,
-    clientId?: string,
+    configuration: {
+      identityClient: IdentityClient;
+      resource: string;
+      scopes: string | string[];
```

Comment:
Does this mean it's valid to call getToken with values for both `resource` and `scopes`?

Reply:
nope, this is internal. Resource is essentially the domain of the first `scope` received (no `.default`)